youtube_dl/extractor/indavideo.py

```diff
@@ -9,8 +11,14 @@
 )
 
 
-class IndavideoEmbedIE(InfoExtractor):
```

Comment:
This is **changing name** because you've merged `IndavideoIE` into `IndavideoEmbedIE` (or simply dropped it) and then renamed it to `IndavideoIE` as a result you've **broken** all download archives made with previous versions of youtube-dl. Just do what is asked.